scan/lib/scan/test_command_generator.rb

```diff
@@ -98,8 +98,7 @@ def xcodebuild_log_path
       # Generate destination parameters
       def destination
         unless Scan.cache[:destination]
-          require 'shellwords'
-          Scan.cache[:destination] = [*Scan.config[:destination]].map { |dst| "-destination #{dst.shellescape}" }.join(' ')
+          Scan.cache[:destination] = [*Scan.config[:destination]].map { |dst| "-destination '#{dst}'" }.join(' ')
```

Comment:
I reverted this, since the `xcodebuild` destination parameter requires a string using `'` or `"` 👍 
we join multiple strings together, that look like this
```
xcodebuild -destination 'platform=iOS Simulator,id=70E1E92F-A292-4980-BC3C-7770C5EEFCFD' -destination 'platform=iOS Simulator,id=DD134998-177F-47DA-99FA-D549D9305476'
```
and all those